.changesets/fix_tninesling_undo_auth_changes.md

```diff
@@ -0,0 +1,5 @@
+### Use supergraph schema to extract auth info ([PR #5047](https://github.com/apollographql/router/pull/5047))
+
+Use supergraph schema to extract auth info as auth information may not be available on the query planner's subgraph schemas. This undoes the auth changes made in #4975.
+
+By [@tninesling](https://github.com/tninesling) in https://github.com/apollographql/router/pull/5047
```

Comment:
This changeset was in 1.46.0 and should be removed.

Reply:
Ok, there's only two hold-overs and  [`52bbe1a` (#5180)](https://github.com/apollographql/router/pull/5180/commits/52bbe1ad54a79904bdafc812dfe130290498eb83) fixes that.  Not sure if you have any perspective on what happened here, but I'm interested!